molecule/default/prepare.yml

```
@@ -17,13 +17,14 @@
 - name: Wait for systemd to complete initialization (Fedora)
   hosts: os_Fedora
   tasks:
-    - name: Wait for systemd to complete initialization # noqa 303
+    - name: Wait for systemd to complete initialization # noqa command-instead-of-module
       ansible.builtin.command: systemctl is-system-running
       register: systemctl_status
       until: "'running' in systemctl_status.stdout"
       retries: 30
       delay: 5
       when: ansible_service_mgr == 'systemd'
+      changed_when: "'running' in systemctl_status.stdout"
```

Since the `systemctl is-system-running` command does not actually change anything, I think this line should instead be:
```suggestion
      changed_when: false
```